L1Trigger/TrackerTFP/src/HoughTransform.cc

```diff
@@ -0,0 +1,200 @@
+#include "L1Trigger/TrackerTFP/interface/HoughTransform.h"
+
+#include <numeric>
+#include <algorithm>
+#include <iterator>
+#include <deque>
+#include <vector>
+#include <set>
+#include <utility>
+#include <cmath>
+
+using namespace std;
+using namespace edm;
+using namespace trackerDTC;
+
+namespace trackerTFP {
+
+  HoughTransform::HoughTransform(const ParameterSet& iConfig, const Setup* setup, const DataFormats* dataFormats, int region) :
+    enableTruncation_(iConfig.getParameter<bool>("EnableTruncation")),
+    setup_(setup),
+    dataFormats_(dataFormats),
+    inv2R_(dataFormats_->format(Variable::inv2R, Process::ht)),
+    phiT_(dataFormats_->format(Variable::phiT, Process::ht)),
+    region_(region),
+    input_(dataFormats_->numChannel(Process::ht), vector<deque<StubGP*>>(dataFormats_->numChannel(Process::gp)))
+  {}
+
+  // read in and organize input product
```

Comment:
Add "(fill array input_)" to end of comment

Reply:
done